spec/overview.html

```diff
@@ -23,7 +23,7 @@ <h1>API Overview</h1>
       The ECMAScript 2022 Internationalization API Specification is designed to complement the ECMAScript 2022 Language Specification by providing key language-sensitive functionality. The API can be added to an implementation of the ECMAScript 2022 Language Specification (ECMA-262 13<sup>th</sup> Edition, or successor).
     </p>
     <p>
-      The ECMAScript 2022 Internationalization API Specification provides several key pieces of language-sensitive functionality that are required in most applications: String comparison (collation), number formatting, date and time formatting, display names, list formatting, pluralization rules, and case conversion. While the ECMAScript 2022 Language Specification provides functions for this basic functionality (on <emu-xref href="#sec-array.prototype">Array.prototype</emu-xref>: `toLocaleString`; on <emu-xref href="#sec-string.prototype">String.prototype</emu-xref>: `localeCompare`, `toLocaleLowerCase`, `toLocaleUpperCase`; on <emu-xref href="#sec-number.prototype">Number.prototype</emu-xref>: `toLocaleString`; on <emu-xref href="#sec-date.prototype">Date.prototype</emu-xref>: `toLocaleString`, `toLocaleDateString`, and `toLocaleTimeString`), it leaves the actual behaviour of these functions largely up to implementations to define. The ECMAScript 2022 Internationalization API Specification provides additional functionality, control over the language and over details of the behaviour to be used, and a more complete specification of required functionality.
+      The ECMAScript 2022 Internationalization API Specification provides several key pieces of language-sensitive functionality that are required in most applications: String comparison (collation), number formatting, date and time formatting, relative time formatting, display names, list formatting, pluralization rules, and case conversion. While the ECMAScript 2022 Language Specification provides functions for this basic functionality (on <emu-xref href="#sec-array.prototype">Array.prototype</emu-xref>: `toLocaleString`; on <emu-xref href="#sec-string.prototype">String.prototype</emu-xref>: `localeCompare`, `toLocaleLowerCase`, `toLocaleUpperCase`; on <emu-xref href="#sec-number.prototype">Number.prototype</emu-xref>: `toLocaleString`; on <emu-xref href="#sec-date.prototype">Date.prototype</emu-xref>: `toLocaleString`, `toLocaleDateString`, and `toLocaleTimeString`), it leaves the actual behaviour of these functions largely up to implementations to define. The ECMAScript 2022 Internationalization API Specification provides additional functionality, control over the language and over details of the behaviour to be used, and a more complete specification of required functionality.
```

Comment:
We may also mention Intl.Locale by adding something like "locale selection and operation" together with 

> ...String comparison (collation), number formatting, date and time formatting, relative time formatting, display names, list formatting, pluralization rules ...

Reply:
Sounds good!

Reply:
Just updated with `locale selection and operation`